modules/ROOT/pages/manage-exchange-api-task.adoc

```diff
@@ -44,13 +52,49 @@ For example, if an API is defined with the `Implementation URI` set to `http://j
 |`/` | `https://domain.cloudhub.io/users` | `http://jsonplaceholder.typicode.com/users`
 | `/api` | `https://domain.cloudhub.io/api/users` | `http://jsonplaceholder.typicode.com/users`
 |===
-. Optionally, expand the *Advanced options* section and specify additional details:
-* Specify the validations and an API instance label for the API.
-* If you are creating an endpoint with proxy, also specify a scheme (HTTP or HTTPS) and a port, or accept the defaults.
-* Specify a response timeout, if required. 
-* If you choose *Hybrid* as the proxy deployment target, select the *Reference user domain* checkbox to use an API gateway domain.
-+
-You must install the API gateway domain in Mule 3.8 and later.
-+
-. Click *Save*.
+. Optionally, to fine-tune additional configurations for the API, expand *Advanced options*, specify the <<field-reference,details>>, and click *Save*.
+
+[[field-reference]]
+== Field Input Reference
+
+The input that you are required to provide when adding an API from Exchange depends on your specific situation, such as whether you want to create an endpoint with proxy, or whether the API was created using Mule 3 or Mule 4:
+
+[%header%autowidth.spread,cols="15%,35%,15%,35%"]
+|===
+| Field Name | Description | Required | Notes
+| *Implementation URI* | The URL to access for the proxy or the API. | Yes|  For example, you can use the URL of your API asset in Exchange. In this case, the API is consuming the JSONPlaceholder API, so enter `+http://jsonplaceholder.typicode.com+`.
+| *Proxy version* | Specifies the version of the proxy to use for the endpoint.| No |
+| *Enable Console* | Specifies whether you can simulate your API specification. | No | You can specify a different path in *Console Path*, for example, "/spec/*".
```

Comment:
Same comment, maybe I would replace "simulate" by "expose and test"

Reply:
Done! Thanks!